### PR TITLE
Add "Alt" Text Attribute to Image

### LINKED ISSUE #594: Audit all img and svg for missing alt text
During user testing, one user noticed that we don't have alt text on our SVG for the certificate pending page.  Let's fix that, and also update all other img/svg uses.

### PR DESCRIPTION
**Pull request by Ririio**
Fixes #594

### CODE REVIEW

File: app/components/display-page.tsx [hunk 1]
```diff
@@ -3,9 +3,10 @@ import { Center, Flex, Image, Heading } from '@chakra-ui/react';
 interface DisplayPageProps {
   img: string;
   desc: string;
+  alt: string;
 }
 
-export default function DisplayPage({ img, desc }: DisplayPageProps) {
+export default function DisplayPage({ img, desc, alt }: DisplayPageProps) {
```
Comment (humphd) [inline app/components/display-page.tsx:9]: While you are touching this file, can you please rename it?  "DisplayPage" doesn't tell you anything about what this is for.

File: app/components/header.tsx [hunk 2]
```diff
@@ -96,24 +96,25 @@ export default function Header() {
         </Heading>
       </Link>
 
-      <Flex justifyContent="flex-end" alignItems="center" color="white" gap="5" width="100%">
```
Comment (humphd) [inline app/components/header.tsx:99]: This change belongs to another PR and shouldn't be in this one.

File: app/routes/_auth.certificate._index.tsx [hunk 3]
```diff
@@ -115,6 +115,7 @@ export default function CertificateIndexRoute() {
       <Loading
         img={pendingSvg}
         desc="We have received your request, and will notify you when your certificate is ready"
+        alt="loading-page-svg"
```
Comment (humphd) [inline app/routes/_auth.certificate._index.tsx:118]: Alt text should describe the image.

**Review by humphd — CHANGES_REQUESTED** [review]

File: app/components/header.tsx (hunk 2 above)
Comment (Ririio) [reply app/components/header.tsx:99]: oh that's odd, why is it here...

File: app/routes/_auth.certificate._index.tsx [hunk 4]
```diff
@@ -115,6 +115,7 @@ export default function CertificateIndexRoute() {
       <Loading
         img={pendingSvg}
         desc="We have received your request, and will notify you when your certificate is ready"
+        alt="two-people-processing"
```
Comment (humphd) [inline app/routes/_auth.certificate._index.tsx:118]: Don't use `-`, it's supposed to be natural language.  Maybe, "Two people at a computer processing a document together"

File: app/components/display-message.tsx [hunk 5]
```diff
@@ -1,11 +1,12 @@
...
 
-interface DisplayPageProps {
+interface DisplayMessageProps {
   img: string;
   desc: string;
+  alt: string;
 }
 
-export default function DisplayPage({ img, desc }: DisplayPageProps) {
+export default function DisplayMessage({ img, desc, alt }: DisplayMessageProps) {
```
Comment (humphd) [inline app/components/display-message.tsx:9]: This name is no better.  What you have here is a component for showing an image with a description.  `ImageWithMessage`?

**Review by humphd — CHANGES_REQUESTED** [review]

File: app/routes/_auth.certificate._index.tsx (hunk 4 above)
Comment (Ririio) [reply app/routes/_auth.certificate._index.tsx:118]: oh so it's literally to describe them alright I'll keep that in mind

**Review by humphd — APPROVED** [review]

**Review by Genne23v — APPROVED** [review]
